Student/StudentUnitTests/RoutesTests.swift

```diff
@@ -17,47 +17,41 @@
 //
 
 import XCTest
-import Core
+@testable import Core
+@testable import CanvasCore
 @testable import Student
 import TestsFoundation
 
 class RoutesTests: XCTestCase {
-    func testLogin() {
-        XCTAssert(router.match(Route.login.url) is LoginNavigationController)
-    }
-
     func testCourses() {
-        XCTAssert(router.match(Route.courses.url) is CourseListViewController)
+        XCTAssert(router.match(Route.courses.url) is HelmViewController)
```

Comment:
not sure if it's worth testing, but would it be good to test the `moduleName` as since HelmViewController is kind of general?

Reply:
These tests are a holdover from the previous Routes.swift. I just did the minimum to un-break them. Should I spend some time making more robust tests for all the routes?

Reply:
It might make sense b/c a HelmViewController could be one of many routes.  Checking the moduleName might seem more like checking the type of ViewController when they it is one route to one type of viewController.